### PR TITLE
Add a link to the OpenShift Console to the web-app

### PR DESCRIPTION
**Pull request by witmicko**
Card: https://trello.com/c/IQeLG5Pj/160-add-a-link-to-the-openshift-console-to-the-web-app

image

@aidenkeating @jasonmadigan

### CODE REVIEW

File: src/components/installedAppsView/InstalledAppsView.js [hunk 1]
```diff
@@ -34,6 +34,18 @@ class InstalledAppsView extends React.Component {
...
   }
 
+  static getOpenshiftConsole(index) {
+    return (
+      <li
+        onClick={() => window.open(`${window.OPENSHIFT_CONFIG.masterUri}/console`, '_blank')}
+        key={`openshift_console_${index}`}
+        value={index}
+      >
+        Openshift Console
```
Comment (jasonmadigan) [inline src/components/installedAppsView/InstalledAppsView.js]: small thing: `Openshift Console` -> `OpenShift Console`

File: src/components/installedAppsView/InstalledAppsView.js [hunk 2]
```diff
@@ -34,6 +34,18 @@ class InstalledAppsView extends React.Component {
     return null;
   }
 
+  static getOpenshiftConsole(index) {
+    return (
+      <li
+        onClick={() => window.open(`${window.OPENSHIFT_CONFIG.masterUri}/console`, '_blank')}
```
Comment (aidenkeating) [inline src/components/installedAppsView/InstalledAppsView.js]: It's possible that `OPENSHIFT_CONFIG` does not contain a `masterUri` key if it's running in mock mode. We need to add something into `server.js` to include this.

Comment (witmicko) [reply src/components/installedAppsView/InstalledAppsView.js]: updated

Comment (dlabaj) [reply src/components/installedAppsView/InstalledAppsView.js]: @witmicko @aidenkeating Has the server.js been updated?  Didn't see it in this review.

File: src/components/installedAppsView/InstalledAppsView.js (hunk 1 above)
Comment (witmicko) [reply src/components/installedAppsView/InstalledAppsView.js]: @jasonmadigan this is fixed now

**Pull request merged**